### PR TITLE
Tmedia 592 hide gallery title

### PR DESCRIPTION
**Pull request by JackHowa**
## Description
Hide options for gallery

## Jira Ticket
- [TMEDIA-592](https://arcpublishing.atlassian.net/browse/TMEDIA-592)

## Acceptance Criteria
Allow hide and show functionality for gallery block

## Test Steps
- Open storybook 
- Go to the hide and show story for gallery block
- Toggle and untoggle the caption, title and credit options 

Screen Shot 2021-12-21 at 14 20 48
Screen Shot 2021-12-21 at 14 20 41

### Before

Gallery block forced users to show all title, caption and credit 

### After
Gallery block allows toggling of the content

## Dependencies or Side Effects
- None. the engine theme sdk already had this feature done

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [ ] Confirmed this PR has unit test files
  - [ ] Ran `npm run test`, made sure all tests are passing
  - [ ] If the amount of work to write unit tests for this change are excessive, 
please explain why (so that we can fix it whenever it gets refactored). -> not applicable change
- [x] Confirmed relevant documentation has been updated/added. -> updated sb

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/gallery-block/features/gallery/default.jsx [hunk 1]
```diff
@@ -101,10 +107,25 @@ GalleryFeature.propTypes = {
...
+    hideCredits: PropTypes.bool.tag({
+      label: 'Hide Credits',
+      defaultValue: false,
+      group: 'Display Options',
+    }),
   }),
 };
 
-GalleryFeature.label = 'Gallery – Arc Block';
+GalleryFeature.label = 'Gallery - Arc Block';
```
Comment (JackHowa) [inline blocks/gallery-block/features/gallery/default.jsx:128]: unrelated fix, my editor didn't recognize whatever symbol "-" was here before

File: blocks/gallery-block/index.story.jsx [hunk 2]
```diff
@@ -99,3 +101,26 @@ export const withPassedInContent = () => {
     />
   );
 };
+
+export const withHiddenContentOptions = () => {
```
Comment (matthewroach) [inline blocks/gallery-block/index.story.jsx]: Shouldn't this story have the values set to true so we can have visual tests showing the items not displayed?

**Review by matthewroach — CHANGES_REQUESTED** [review]
All works as expected from code perspective - but the story is a little mis-leading for visual regression testing, I see the use of knobs to allow users to toggle the options to see how the block works, but from a testing perspective we want to ensure we capture the different visual states

File: blocks/gallery-block/index.story.jsx (hunk 2 above)
Comment (JackHowa) [reply blocks/gallery-block/index.story.jsx]: updated

**Review by matthewroach — APPROVED** [review]
